info-compiler/src/main/java/com/google/sps/webcrawler/NewsContentProcessor.java

```diff
@@ -19,7 +19,7 @@
 
 /** Static utilities for processing textual content, such as abbreviation. */
 public class NewsContentProcessor {
-  public final static int MAX_WORD_COUNT = 100;
+  default static final int MAX_WORD_COUNT = 100;
```

Comment:
*Ah sorry you can leave off the word "default" for these (methods & variables) and it should have the same level of visibility.
```suggestion
  static final int MAX_WORD_COUNT = 100;
```

Reply:
Oops looks like you did this already.

Reply:
Thanks for the reminder! I just realized too and removed those